ftplugin/python/pytest.vim

```diff
@@ -896,7 +896,7 @@ function! s:ThisMethod(verbose, ...)
       return
     endif
     if ((a:1 == '--pdb') || (a:1 == '-s'))
-        call s:Pdb(path, a:1)
+        call s:Pdb(path, a:1, m_name)
```

Comment:
Can anything be changed to simplify the parameterized check later in `s:Pdb`? as I copied the way it was handled in `s:RunPyTest` (apart from incorrectly using `m_name` instead of `parameterized_flag`)